bot/exts/utils/snekbox.py

```diff
@@ -344,8 +344,14 @@ async def send_job(
                 log.trace("Formatting output...")
                 output, paste_link = await self.format_output(results["stdout"])
 
+            warning_message = ""
+            # 33 is the length of the error message. It is done to make sure the last line of output contains
+            # the error and the error is not manually printed by the author with a syntax error.
+            if "EOFError: EOF when reading a line" in output[-33:] and results['returncode'] == 1:
+                warning_message += ":warning: Note: `input` is not supported by the bot :warning:\n\n"
```

Comment:
```suggestion
            # the error and the error is not manually printed by the author with a syntax error.
            if results["stdout"].rstrip().endswith("EOFError: EOF when reading a line") and results["returncode"] == 1:
                warning_message += ":warning: Note: `input` is not supported by the bot :warning:\n\n"
```
I'd suggest:
- Double quotes for consistency
- Using `results["stdout"]` instead of `output` so the warning is still displayed if the error doesn't fit in the message.
- Using `endswith` so we don't need to hardcode the constant.